<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMatplotlibSubfigConfig</class>
 <widget class="QDialog" name="frmMatplotlibSubfigConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subplot Configuration Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Margins</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>left</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="DoubleRangeWidget" name="dblRangeLeftRight">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="maximumValue">
         <double>100.000000000000000</double>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>right</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>bottom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="DoubleRangeWidget" name="dblRangeBottomTop">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="maximumValue">
         <double>100.000000000000000</double>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>top</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Spacing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>vertical space</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>horizontal space</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="SliderWidget" name="sliderWSpace">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="SliderWidget" name="sliderHSpace">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleRangeWidget</class>
   <extends>QWidget</extends>
   <header>doubleRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SliderWidget</class>
   <extends>QWidget</extends>
   <header>sliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>frmMatplotlibSubfigConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>frmMatplotlibSubfigConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
